README.md

```diff
@@ -31,51 +28,103 @@ Imagine a declarative Kubernetes-style API for _anything_, supported by an ecosy
 That's **`kcp`**.
 
 
-## Is `kcp` a "fork" of Kubernetes? 🍴
+## Can I start using `kcp` today?
 
-_No._
+Yes! This work is still in early development, which means it's _not ready for production_ and _APIs, commands, flags, etc. are subject to change_, but also that your feedback can have a big impact. Please try it out and [let us know](#this-sounds-cool-and-i-want-to-help) what you like, dislike, what works, what doesn't, etc.
 
-`kcp` as a prototype currently depends on some unmerged changes to Kubernetes, but we intend to pursue these changes through the usual KEP process, until (hopefully!) Kubernetes can be configured to run as `kcp` runs today.
+`kcp` is currently a prototype, not a project. We're exploring these ideas here to try them out, experiment, and bounce them off each other.
 
-Our intention is that our experiments _improve Kubernetes for everyone_, by improving [CRD](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)s and scaling resource watching, and enabling more, better controllers _for everyone_, whether you're using Kubernetes as a container orchestrator or not.
 
-Our `kcp` specific patches are in the [feature-logical-clusters-1.22](https://github.com/kcp-dev/kubernetes/tree/feature-logical-clusters-1.22) feature branch in the [kcp-dev/kubernetes](https://github.com/kcp-dev/kubernetes) repo. See [DEVELOPMENT.md](DEVELOPMENT.md) for how the patches are structured and how they must be formatted during our experimentation phase.  See [GOALS.md](GOALS.md) for more info on how we intend to use `kcp` as a test-bed for exploring ideas that improve the entire ecosystem.
+## Can `kcp` do anything else?
 
+Yes! Here are a few of our top-level goals:
 
-## What's in this repo?
+### Host thousands (*) of small-ish Kubernetes-like [logical clusters](docs/investigations/logical-clusters.md) in a single instance
 
-First off, this is a prototype, not a project. We're exploring these ideas here to try them out, experiment, and bounce them off each other.  Our [basic demo](contrib/demo/README.md) leverages the following components to show off these ideas:
+* Orders of magnitude smaller than a typical large, shared, multi-tenant cluster (~50 resources per cluster)
+* Inexpensive - nearly for free in terms of resource utilization & cost for an empty cluster
+* Independent - install different versions of a CRD in each logical cluster (!!!)
+* Per-cluster administrative rights - each "owner" (person/team/group) of a cluster is a full admin
```

Comment:
That's good, as far as it goes.  But since kcp introduces a higher level of organization, it also must involve higher level operations and thus higher level authorization issues.  What about those?

If I understand correctly, the compute clusters used by the workload service are a separate matter from the workspaces of kcp.  That itself raises some more authentication and authorization questions.